### PR TITLE
license acceptance idling changes

### PR DESCRIPTION
**Pull request by bitscuit**

### CODE REVIEW

File: controllers/commonservice_controller.go [hunk 1]
```diff
@@ -97,6 +97,16 @@ func (r *CommonServiceReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	masterCR := &apiv3.CommonService{}
+	if err := r.Bootstrap.Client.Get(ctx, types.NamespacedName{Namespace: r.Bootstrap.CSData.OperatorNs, Name: "common-service"}, masterCR); err != nil {
+		return ctrl.Result{}, err
+	}
+
+	if !masterCR.Spec.License.Accept {
+		klog.Info("Accept license by changing .spec.license.accept to true in the CertManagerConfig CR. Operator will not proceed until then")
```
Comment (bluzarraga) [inline controllers/commonservice_controller.go:106]: Should this info line be referencing the commonservice CR instead of the CertManagerConfig CR?

Comment (bitscuit) [reply controllers/commonservice_controller.go:106]: yes, good catch

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**